### PR TITLE
[log] print a message in validator if consensus reached

### PR DESCRIPTION
**Pull request by LeoHChen**
This will help the external node runner to check on their log.

Signed-off-by: Leo Chen <leo@harmony.one>

### CODE REVIEW

**Comment by LeoHChen** [comment]
Jenkins, test this please.

File: node/node_handler.go [hunk 1]
```diff
@@ -281,7 +281,10 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block) {
 	utils.GetLogInstance().Info("PostConsensusProcessing")
 	if node.Consensus.IsLeader {
 		node.BroadcastNewBlock(newBlock)
+	} else {
+		utils.GetLogInstance().Info("!!! BINGO !!!", "Reached Consensus", node.Consensus.GetConsensusID())
```
Comment (lzl124631x) [inline node/node_handler.go]: nit: "reached consensus" is used as the key to `consensusID` which is not as straightforward as "ConsensusID" and moving "Reached Consensus" to after "Bingo"

Comment (LeoHChen) [reply node/node_handler.go]: sure. changed.

File: node/node_handler.go [hunk 2]
```diff
@@ -281,7 +281,10 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block) {
 	utils.GetLogInstance().Info("PostConsensusProcessing")
 	if node.Consensus.IsLeader {
 		node.BroadcastNewBlock(newBlock)
+	} else {
+		utils.GetLogInstance().Info("BINGO !!! Reached Consensus", "ConsensusID", node.Consensus.GetConsensusID())
 	}
```
Comment (alajko) [inline node/node_handler.go]: Is there a validator related ID we can include here. Else we will have a flood of messages with same `consensusID`? consensusID is always `0`?

Comment (LeoHChen) [reply node/node_handler.go]: consensusID change after each consensus.  This is per validator message, it won't flood.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Pull request merged**